baseapp/baseapp.go

```diff
@@ -56,20 +57,30 @@ var _ abci.Application = (*BaseApp)(nil)
 // Create and name new BaseApp
 // NOTE: The db is used to store the version number for now.
 func NewBaseApp(name string, logger log.Logger, db dbm.DB) *BaseApp {
-	return &BaseApp{
-		Logger: logger,
-		name:   name,
-		db:     db,
-		cms:    store.NewCommitMultiStore(db),
-		router: NewRouter(),
+	app := &BaseApp{
+		Logger:     logger,
+		name:       name,
+		db:         db,
+		cms:        store.NewCommitMultiStore(db),
+		router:     NewRouter(),
+		codespacer: sdk.NewCodespacer(),
 	}
+	// Register the undefined & root codespaces, which should not be used by any modules
+	app.codespacer.RegisterNext(sdk.CodespaceUndefined)
```

Comment:
For these we shouldn't use RegisterNext.  Shouldn't be using it unless we're going to use the return value, basically.  Just Register() and check error or maybe it panics, I forget.

Reply:
Changed to `RegisterOrPanic`